lib/shelley/src/Cardano/Wallet/Shelley/Compatibility.hs

```diff
@@ -29,14 +30,13 @@
 -- License: Apache-2.0
 --
 -- Conversion functions and static chain settings for Shelley.
-
 module Cardano.Wallet.Shelley.Compatibility
     ( CardanoBlock
     , StandardCrypto
     , StandardShelley
 
       -- * Protocol Parameters
-    , NetworkId (..)
+    , NetworkId(..)
```

Comment:
```suggestion
    , NetworkId (..)
```

Uh, could you add the space again? It's our current coding style convention. 🤷 Similar for the other exports. (The imports follow the same pattern)

Reply:
cannot find this difference ...

Reply:
Done!